Source/VerveAdapterBannerAd.swift

```diff
@@ -38,7 +38,7 @@ final class VerveAdapterBannerAd: VerveAdapterAd, PartnerAd {
         // Load differently depending on whether this is a bidding or non-programatic ad
         if let adm = request.adm {
             ad.delegate = self
-            ad.prepareCustomMarkup(from: adm)
+            ad.prepareCustomMarkup(from: adm, with: HyBidDemoAppPlacementBanner)
```

Comment:
Can you explain why we need to use a demo app placement here?
If we must use it, can you add a comment clarifying its purpose?

Reply:
The new method signature is `- (void)prepareCustomMarkupFrom:(NSString *)markup withPlacement:(HyBidMarkupPlacement)placement`.  The previous one no longer exists.  The enum `HyBidMarkupPlacement` is:

```
typedef enum {
    HyBidDemoAppPlacementBanner = 0,
    HyBidDemoAppPlacementMRect = 1,
    HyBidDemoAppPlacementLeaderboard = 2,
    HyBidDemoAppPlacementInterstitial = 3,
    HyBidDemoAppPlacementRewarded = 4
} HyBidMarkupPlacement;
```

All of them have demo in their name for some reason.

Reply:
BTW, this is all I had found about their banners API: https://developers.verve.com/reference/hybid-ios-sdk-standalone-banner-ads?__hstc=143843498.079b895e5ffbe3031d958b14fe6bc233.1708649505027.1708649505027.1708649505027.1&__hssc=143843498.1213.1708649505027&__hsfp=2383419112

I can't find anything anywhere about `prepareCustomMarkupFrom` and there is no documentation in their headers.

Reply:
so weird. can you add a comment clarifying that this is not actually a "demo app" placement, but more like an "ad format", and the "demo" thing is a strange naming decision from HyBid?
It'll be less confusing for new people reading this line. Thanks

Reply:
I found their Wiki documentation on GitHub.  Same nothingness.

https://github.com/pubnative/pubnative-hybid-ios-sdk/wiki/HyBid-Banner-Ads

How did whoever made this adapter figure anything out initially?  Some secret documentation somewhere?

Reply:
lol I don't know. It was probably copied from the old 3.x Helium adapters... let's just go with what we have I guess

Reply:
@daniel-barros From the changes Verve did in https://github.com/ChartBoost/chartboost-mediation-ios-adapter-verve/pull/8, this original change is no longer applicable.  It is now

```
ad.renderAd(withContent: adm, with: self)
```